cogs/ReactForRole.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, bot: discord.Client):
         self.rfr_database_manager = ReactForRoleDBManager(KoalaBot.database_manager)
         self.rfr_database_manager.create_tables()
 
+    @commands.check(KoalaBot.is_guild_channel)
```

Comment:
```suggestion
    @commands.check(KoalaBot.is_guild_channel)
    @commands.check(KoalaBot.is_admin)
    @commands.check(rfr_is_enabled)
```
needs check so it doesn't show in k!help for everyone